### PR TITLE
Update HTML snippet

### PR DESCRIPTION
**Pull request by afeijoo**
Add language and agency variables to standard script. Update instructions to point to implementation guide for custom params.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,12 +15,12 @@ Agencies are encouraged to use the following HTML snippet to participate in the
...
-<script src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js" id="_fed_an_ua_tag"></script>
+<script language="javascript" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=AGENCY" id="_fed_an_ua_tag"></script>
 ```
 
-For more details on implementing the DAP script on your site, please refer to the [DAP Implementation Instructions](http://www.digitalgov.gov/services/dap/analytics-tool-instructions/) and the [Implementation Guide](https://www.digitalgov.gov/files/2015/02/GSA-DAP-UA-Code-Quick-Guide-15-01-30-v1-02_mvf.pdf).
+Replace `AGENCY` with your agency's	standard	abbreviation, such as DHS or EPA.
 
-Please note that there are required parts of the script tag syntax, such as `id=_fed_an_ua_tag` as well as `agency` & `subagency` parameters.
+For more details on implementing the DAP script on your site, including adding other custom parameters, please refer to the [DAP Implementation Instructions](http://www.digitalgov.gov/services/dap/analytics-tool-instructions/) and the [Implementation Guide](https://www.digitalgov.gov/files/2015/02/GSA-DAP-UA-Code-Quick-Guide-15-01-30-v1-02_mvf.pdf).
```
Comment (konklone) [inline README.md]: the first `http://www.digitalgov.gov` link on this line can use `https://`.

File: README.md [hunk 2]
```diff
@@ -15,12 +15,12 @@ Agencies are encouraged to use the following HTML snippet to participate in the
 
 ```html
 <!-- We participate in the US government's analytics program. See the data at analytics.usa.gov. -->
-<script src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js" id="_fed_an_ua_tag"></script>
+<script language="javascript" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=AGENCY" id="_fed_an_ua_tag"></script>
 ```
 
-For more details on implementing the DAP script on your site, please refer to the [DAP Implementation Instructions](http://www.digitalgov.gov/services/dap/analytics-tool-instructions/) and the [Implementation Guide](https://www.digitalgov.gov/files/2015/02/GSA-DAP-UA-Code-Quick-Guide-15-01-30-v1-02_mvf.pdf).
+Replace `AGENCY` with your agency's	standard	abbreviation, such as DHS or EPA.
```
Comment (konklone) [inline README.md]: There's two tabs in the middle of this line creating extra spacing between words.

File: README.md [hunk 3]
```diff
@@ -15,12 +15,12 @@ Agencies are encouraged to use the following HTML snippet to participate in the
 
 ```html
 <!-- We participate in the US government's analytics program. See the data at analytics.usa.gov. -->
-<script src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js" id="_fed_an_ua_tag"></script>
+<script language="javascript" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=AGENCY" id="_fed_an_ua_tag"></script>
```
Comment (konklone) [inline README.md]: `language="javascript"` is optional in all browsers, I'm pretty sure -- but I can see why you'd want to keep it consistent with the implementation guide. So consider that a note for the implementation guide. =)

**Comment by konklone** [comment]
I left a few comments inline above - it'd be nice to switch the one `http://` protocol to `https://` before merging.

File: README.md (hunk 1 above)
Comment (tdlowden) [inline README.md]: @afeijoo can you adjust?

File: README.md (hunk 2 above)
Comment (tdlowden) [inline README.md]: @afeijoo after we make these tiny adjustments, i'll merge

**Comment by konklone** [comment]
@afeijoo updated it and :+1: from me.

**Pull request merged**